examples/endpoint_unity_example/readme.md

```diff
@@ -0,0 +1,10 @@
+# Octupus Sensing - Unity Example
+In this example, We're going to send predefined messages (START, STOP and, TERMINATE) from Unity to Octopus sensing, to control the recording of data. Octopus sensing (Python) is used as a server and Unity (C#) as a client.
+
```

Comment:
You are not using Octopus Sensing as the server. Octopus Sensing is a library. You have a server in python developed using the Octopus Sensing library.